concent_api/conductor/service.py

```diff
@@ -54,3 +59,50 @@ def update_upload_report(file_path: str, result_transfer_request: ResultTransfer
         result_transfer_request.save()
 
         result_upload_finished.delay(result_transfer_request.subtask_id)
```

Comment:
This task is being posted still inside `@atomic` while the changes have not been committed yet.

Reply:
:+1: done